CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/plugins/GettingStartedWelcomePlugin.java

```diff
@@ -0,0 +1,125 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.welcome.plugins;
+
+import au.gov.asd.tac.constellation.views.welcome.WelcomePageProvider;
+import au.gov.asd.tac.constellation.views.welcome.WelcomeTopComponent;
+import au.gov.asd.tac.constellation.plugins.PluginInfo;
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import org.openide.util.NbBundle;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ * The plugin for the Welcome Page that leads to the Getting Started guides and resources
+ *
+ * @author Delphinus8821
+ */
+
+@ServiceProvider(service = WelcomePageProvider.class, position = 5)
+@PluginInfo(tags = {"WELCOME"})
+@NbBundle.Messages("GettingStartedWelcomePlugin=Getting Started Welcome Plugin")
+public class GettingStartedWelcomePlugin extends WelcomePageProvider {
+    
+    public static final String GETTING_STARTED = "resources/welcome_getting_started.png";
+    final ImageView started = new ImageView(new Image(WelcomeTopComponent.class.getResourceAsStream(GETTING_STARTED)));
+    final Button startedBtn = new Button();
+    
+    private static final Logger LOGGER = Logger.getLogger(GettingStartedWelcomePlugin.class.getName());
+        
+    /**
+     * Get a unique reference that is used to identify the plugin 
+     *
+     * @return a unique reference
+     */
+    @Override
+    public String getName() {
+        return "Getting Started Welcome";
+    }
+    
+    /**
+     * This method describes what action should be taken when the 
+     * link is clicked on the Welcome Page
+     *
+     */
+    @Override
+    public void run() {
+        String url = "https://constellation.readthedocs.io/en/latest/";
+
+        if(Desktop.isDesktopSupported()){
+            Desktop desktop = Desktop.getDesktop();
+            try {
+                desktop.browse(new URI(url));
+            } catch (IOException | URISyntaxException e) {
+                LOGGER.log(Level.WARNING, e.getMessage());
+            }
+        } else {
+            Runtime runtime = Runtime.getRuntime();
+            try {
+                runtime.exec("xdg-open " + url);
+            } catch (IOException e) {
+                LOGGER.log(Level.WARNING, e.getMessage());
+            }
```

Comment:
I'm wondering if we want this or whether we not bother. If we can't open the link in a browser from what the `Desktop` utility provides then I feel we shouldn't worry about running it from the command line like this. If you agree, can we remove these blocks from the different `WelcomeProvider` plugins?